scripts/simplesamlphp/acquia_config.php

```diff
@@ -21,8 +21,13 @@
 // Change these for your installation
 $config['secretsalt'] = 'y0h9d13pki9qdhfm3l5nws4jjn55j6hj';
 $config['auth.adminpassword'] = 'mysupersecret';
-// Prevent Varnish from interfering with SimpleSAMLphp.
-setcookie('NO_CACHE', '1');
```

Comment:
@kylebrowning I remember at one point seeing requests to `/saml_login/*` being cached unintentionally in Varnish and causing issue. Have we verified this works as expected? The NO_CACHE cookie could also be scoped to a specific path.